kdenlive/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "2.1"
+
+services:
+  server:
```

Comment:
Could you add the Umbrel App Proxy docker service here?

Reply:
Using `app_proxy` didn't work for me. It was showing the port already existed at the time of making the PR. Will try again now.

Reply:
pushed changes for this as well. Sorry. My bad

Reply:
This could be because the app proxy and the `server` service were trying to open the same port.

Reply:
Yeah. Right